docs/prime-number.md

```diff
@@ -0,0 +1,23 @@
+# Prime Numbers
+
+A prime number is a positive integer which is divisible only by 1 and itself.
+For example: 2, 3, 5, 7, 11, 13
+
+Since every number is divisible by One so, Two is the only even and the
+smallest prime number.
+
+
+## Requirements
+
+Create a file called Prime Number using the naming
+convention appropriate for your language of choice.
+
+Write a sample program which accepts an integer on the command line
+and outputs if the integer is a Prime number or not.
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```

Comment:
For prime, we could include a table that has the same test cases as even/odd. One prime number, one not prime number, zero, and a negative number. It may be good to include a test case for empty input as well.